demos/drawer/dismissible-drawer.html

```diff
@@ -43,44 +43,62 @@ <h6 class="mdc-drawer__subtitle">email@material.io</h6>
         <nav class="mdc-list">
           <a class="mdc-list-item mdc-list-item--activated" href="#" aria-selected="true" tabindex="0">
             <i class="material-icons mdc-list-item__graphic" aria-hidden="true">inbox</i>
-            Inbox
+            <span class="mdc-list-item__text">
```

Comment:
Since we restructured this can you change `event.target.lastChild.textContent` to `event.target.lastElementChild.textContent` in all drawer demos?